### PR TITLE
separate sponsors into tiers

### PR DESCRIPTION
**Pull request by alvinkam33**
idk how this will look with the actual sponsors, but if there is an issue i can revisit. lmk if i should add a header for each of the tiers

### CODE REVIEW

**Review by AbdullahWins — COMMENTED** [review]
Well Done!

File: src/sections/Sponsor.jsx [hunk 1]
```diff
@@ -139,45 +157,56 @@ const Skip = styled.div`
...
       <Sponsors>
-        {sponsors.map((sponsor) => (
-          <SponsorLogo src={sponsor.imgURL}/>
+        <StyledTitle>
+          Sponsors
+        </StyledTitle>
+        {Object.keys(sponsors).sort((a, b) => sponsorTierOrder[a] - sponsorTierOrder[b]).map((key) => (
+          <SponsorTier>
+            {sponsors[key].map((sponsor) => (
+              <SponsorLogo src={sponsor.imgURL}/>
```
Comment (meleongg) [inline src/sections/Sponsor.jsx]: Would it be possible to link to the link contained in `sponsor.link`?
image

Comment (alvinkam33) [reply src/sections/Sponsor.jsx]: yer added

**Review by meleongg — APPROVED** [review]
lgtm

**Pull request merged**